Rizzcomic: Updated domain

Mondotosz opened the pull request:
closes https://github.com/keiyoushi/extensions-source/issues/2590

I've never done mobile development nor worked with Kotlin so it took a while to test it but I built the apk and installed it on my phone to check if it actually works and it does.

Checklist:

- [x] Updated `extVersionCode` value in `build.gradle` for individual extensions
- [x] Updated `overrideVersionCode` or `baseVersionCode` as needed for all multisrc extensions
- [x] Referenced all related issues in the PR body (e.g. "Closes #xyz")
- [x] Added the `isNsfw = true` flag in `build.gradle` when appropriate
- [x] Have not changed source names
- [x] Have explicitly kept the `id` if a source's name or language were changed
- [x] Have tested the modifications by compiling and running the extension through Android Studio
- [x] Have removed `web_hi_res_512.png` when adding a new extension

Mondotosz committed:
Rizzcomic: Updated domain

bapeey commented on src/en/rizzcomic/build.gradle:
```diff
@@ -2,8 +2,8 @@ ext {
     themePkg = 'mangathemesia'
-    baseUrl = 'https://rizzcomic.com'
-    overrideVersionCode = 2
+    baseUrl = 'https://rizzfables.com/'
```
```suggestion
    baseUrl = 'https://rizzfables.com'
```

Mondotosz replied:
I just noticed and pushed a commit addressing that 😅

bapeey commented on src/en/rizzcomic/src/eu/kanade/tachiyomi/extension/en/rizzcomic/RizzComic.kt:
```diff
@@ -23,7 +23,7 @@ import java.util.Locale
 class RizzComic : MangaThemesiaAlt(
     "Rizz Comic",
-    "https://rizzcomic.com",
+    "https://rizzfables.com/",
```
```suggestion
    "https://rizzfables.com",
```

bapeey reviewed (commented):
Remove trailing slash

Mondotosz committed:
fix: removed trailing slashes

it didn't have any impact in my tests but it wasn't present before so it's probably a good idea to remove it
